editor/src/components/navigator/layout-element-icons.ts

```diff
@@ -75,24 +76,22 @@ export function createLayoutOrElementIconResult(
 ): LayoutIconResult {
   let hasWidthOrHeight: boolean = false
 
-  const element = TP.isInstancePath(path)
-    ? MetadataUtils.getElementByInstancePathMaybe(metadata, path)
-    : null
+  const element = MetadataUtils.findElementByTemplatePath(metadata, path)
 
   if (element != null && element.props != null && element.props.style != null) {
     hasWidthOrHeight = element.props.style['width'] != null || element.props.style['height'] != null
   }
 
   const layoutIcon = createLayoutIconProps(path, metadata)
-  if (TP.isScenePath(path)) {
+  if (isProbablySceneFromMetadata(metadata, path)) {
     return {
       iconProps: {
         category: 'component',
         type: 'scene',
         width: 18,
         height: 18,
       },
-      hasWidthOrHeight: hasWidthOrHeight,
+      hasWidthOrHeight: false,
```

Comment:
this seems like an unrelated change?

Reply:
Not quite. This would previously always be `false` for a `ScenePath` (and therefore any descendant of the storyboard), but that logic was coming from the deleted line above.